roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -74,6 +74,9 @@ kubernetesMasterConfig:
   masterCount: 1
   masterIP: ""
   podEvictionTimeout: ""
+  proxyClientInfo:
```

Comment:
I agree with @sdodson, I'd like to avoid adding logic to the templates for items that are ignored. Longer term the templates will be going away in favor of a better method to manage configuration.